exportlegends.lua

```diff
@@ -194,6 +194,18 @@ function export_more_legends_xml()
     end
     file:write("</rivers>\n")
 
+    file:write("<creature_raw>\n")
+    for creatureK, creatureV in ipairs (df.global.world.raws.creatures.all) do
+        file:write("\t<creature>\n")
+        file:write("\t\t<creature_id>"..creatureV.creature_id.."</creature_id>\n")
+        file:write("\t\t<name_singular>"..creatureV.name[0].."</name_singular>\n")
+        file:write("\t\t<name_plural>"..creatureV.name[1].."</name_plural>\n")
```

Comment:
Just as a heads-up, it appears that these names can be generated, which means that they need to be escaped (df2utf and probably escape_xml to be safe) to avoid errors like these:
```
$ xmllint --noout medium-world-04702-00162-01-01-legends_plus.xml 
medium-world-04702-00162-01-01-legends_plus.xml:157043: parser error : Input is not proper UTF-8, indicate encoding !
Bytes: 0x98 0x6F 0x6E 0x61
		<name_singular>folly of �onaliitheca</name_singular>
		                        ^
```